pandas/io/tests/test_parsers.py

```diff
@@ -2690,6 +2690,41 @@ def test_uneven_lines_with_usecols(self):
         df = self.read_csv(StringIO(csv), usecols=usecols)
         tm.assert_frame_equal(df, expected)
 
```

Comment:
make another tests that shows ``EmptyDataError``. Furthermore the c-parser needs to conform to this as well.